docs/topics/testing/tools.txt

```diff
@@ -687,6 +687,13 @@ or by including the ``Accept-Language`` HTTP header in the request::
 
 More details are in :ref:`how-django-discovers-language-preference`.
 
+When setting the language for a test client request by either creating a cookie or by including the Accept-Language HTTP header, the Django active language will switch for all other subsequent tests. 
+
+Fortunately, the default language can be set in the tearDown method::
+
+	def tearDown(self):
+		translation.activate(settings.LANGUAGE_CODE)
+
```

Comment:
Docs should be wrapped at 79 chars. See https://docs.djangoproject.com/en/4.1/internals/contributing/writing-code/coding-style/#python-style

I'd rephrase but put it in a callout, before the `More details...` line. 

```suggestion
.. note::

    When using these methods, ensure to reset the active language at the end of
    each test::

        def tearDown(self):
            translation.deactivate()

More details are in :ref:`how-django-discovers-language-preference`.
```